build.psm1

```diff
@@ -15,15 +15,15 @@ $tagsUpToDate = $false
 # When not using a branch in PowerShell/PowerShell, tags will not be fetched automatically
 # Since code that uses Get-PSCommitID and Get-PSLatestTag assume that tags are fetched,
 # This function can ensure that tags have been fetched.
-# This function is used during the setup phase in tools/ci.psm1 and tools/travis.ps1
```

Comment:
It would have been good to combine this with the PR to remove AppVeyor mentions.  This is close enought.

Reply:
Sorry, didn’t want to have to deal with merge conflicts.